### PR TITLE
Add documentation on using GitHub Codespaces

### PR DESCRIPTION
**Pull request by mattstratton**
Connected to #13758

### CODE REVIEW

File: utilities/docs/using-codespaces/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Make sure you push to your fork and not the main repository!
+
+![](img/origin.png)
+
+Now you can create a pull request to merge your changes into the main repository, the same way you would with any other method.
+
+## Stopping your codespace
+
+When you are not using your Codespace, you want to be sure to stop it, so you are not using up your hours. See [Stopping and starting a codespace](https://docs.github.com/en/codespaces/developing-in-a-codespace/stopping-and-starting-a-codespace) for more methods to do this.
+
```
Comment (jerdog) [inline utilities/docs/using-codespaces/README.md:67]: I would suggest adding a link to see all of their running codespaces (https://github.com/codespaces) either in this section or in the billing section so they can stay on top of what is running or what they already have.

Comment (mattstratton) [reply utilities/docs/using-codespaces/README.md:67]: feel free make a suggestion for me to commit!

Comment (mattstratton) [reply utilities/docs/using-codespaces/README.md:67]: I didn't meant that snarky, by the way, more like - "I'd appreciate a suggestion for phrasing"!

Comment (jerdog) [reply utilities/docs/using-codespaces/README.md:67]: LOL - No I get it. I will try to get something added when I'm at my computer

Comment (mattstratton) [reply utilities/docs/using-codespaces/README.md:67]: see https://github.com/devopsdays/devopsdays-web/pull/13760/commits/d34ef7b2bf679b5278ed1ca6e1cb62395dc0825a

**Pull request merged**